### PR TITLE
#274 document file access via JS task

### PR DESCRIPTION
**Pull request by TimKam**

### CODE REVIEW

File: docs/integration/javascript.rst [hunk 1]
```diff
@@ -112,6 +112,31 @@ When clicking `Test it` again, we can see the JSON structure of the variable dat
...
+  const fileContent = files.getContent(contract)
+
+The following example loads a CSV file and parses its content:
+
+.. code:: js
+
+  const files = require('files')
+  const csv = require('csv')
+
+  const csvFile = files.getContent(cSV.id)
```
Comment (SilentGert) [inline docs/integration/javascript.rst]: Sorry, but I just noticed that now. Could you please change `cSV.id` to something more expressive like `reportCsv` and mention that `reportCsv` is an activated workflow variable?

Comment (TimKam) [reply docs/integration/javascript.rst]: Is an `activated workflow` a case/workflow instance?

Comment (SilentGert) [reply docs/integration/javascript.rst]: > activated workflow variable

A variable activated for the script task.

Comment (TimKam) [reply docs/integration/javascript.rst]: Ah, same as in the example above, sure.

Comment (TimKam) [reply docs/integration/javascript.rst]: I'll just add this as a comment to the code snippet

**Pull request merged**